File: driver-portal/src/ClientApp/src/app/shared/api/models/user-registration.ts
/* tslint:disable */
/* eslint-disable */
import { FullAddress } from '../models/full-address';
export interface UserRegistration {
  address?: FullAddress;
  driverLicenseNumber?: string | null;
  email?: string | null;
  notifyByEmail?: boolean;
  notifyByMail?: boolean;
}

File: driver-portal/src/API/ViewModels/UserRegistration.cs
using Rsbc.Dmf.CaseManagement.Service;

namespace Rsbc.Dmf.DriverPortal.ViewModels
{
    public class UserRegistration
    {
        public string DriverLicenseNumber { get; set; }
        public string Email { get; set; }
        public bool NotifyByMail { get; set; }
        public bool NotifyByEmail { get; set; }
        public FullAddress? Address { get; set; }

    }
}
